src/main/java/command/factory/CommandFactory.java

```diff
@@ -0,0 +1,39 @@
+package command.factory;
+
+import command.CommandExecutor;
+import command.add.AddCommand;
+import command.del.DeleteCommand;
+import command.mod.ModifyCommand;
+import command.sch.SearchCommand;
+import model.UserRequest;
+
+public class CommandFactory {
+
+    final private CommandExecutor addCommand;
+    final private CommandExecutor delCommand;
+    final private CommandExecutor modCommand;
+    final private CommandExecutor schCommand;
+    final private CommandExecutor noneCommand;
+
+    public CommandFactory(){
+        addCommand = new AddCommand();
+        schCommand = new SearchCommand();
+        delCommand = new DeleteCommand(schCommand);
+        modCommand = new ModifyCommand(schCommand);
+        noneCommand = new NoneCommand();
+    }
+
+    public CommandExecutor GetCommand(UserRequest request){
```

Comment:
함수명은 카멜기법으로 commanExecutor 수정하는게 가독성 차원에서 어떨까 싶습니다

Reply:
동의합니다~

Reply:
CommandExecutor는 인터페이스입니다
함수명을
```suggestion
    public CommandExecutor getCommand(UserRequest request){
```
이렇게 되는걸 말씀하신거 맞을까요?

Reply:
인터페이스명은 CommandExecutor로 유지하는게 맞아보이고, 함수에 대해 카멜기법을 적용하는게 맞는듯합니다...!